.flake8

```diff
@@ -23,8 +23,6 @@ ignore =
     N806
     # blank line at end of file
```

Comment:
Maybe we can enable blank line check, too?

Reply:
Sure, it can be done in a separate PR. The goal is to remove all exceptions as long as it doesn't conflict with a major design decision.